.github/workflows/release.yml

```diff
@@ -39,29 +39,21 @@ jobs:
           bazel test --local_ram_resources=4096 -c opt -k --test_timeout 300,450,1200,3600 --test_output=errors //tensorflow_recommenders_addons/...
   release-wheel:
     name: Build release wheels
-    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # TODO: add back 'windows-latest' when it can be compiled.
         os: ['macos-12', 'ubuntu-20.04']
         py-version: ['3.7', '3.8', '3.9']
```

Comment:
Please add Python 3.10 for fully matching the TensorFlow releases : https://www.tensorflow.org/install/source#cpu

Reply:
There is no docker image with Python version 3.10, we need to build a one

Reply:
Okay, I will add one after your PR merges, it should be a different goal.

Reply:
I built an image with python 3.10 and uploaded it to the TRFA docker hub.